spec/controllers/mentors_controller_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'rails_helper'
+
+RSpec.describe MentorsController, type: :controller do
+
+  describe "GET #index" do
+    it "returns http success" do
+      get :index
+      expect(response).to have_http_status(:success)
+    end
+  end
+
```

Comment:
awesome 🎉

Reply:
I didn't actually write this, must have been automatic? 

... should I be writing tests for these? I may need your help again getting started ...

Reply:
at this point no.. we don't need to write tests really but when we do we can do a few together